### PR TITLE
add standard footer to all docs pages

### LINKED ISSUE #20: add standardised mysociety footer
In synch with everypolitician/viewer-sinatra#10691

### PR DESCRIPTION
**Pull request by davewhiteland**
resources copied across from everypolitician/viewer-sinatra#10691
closes #20 

Put this live in synch with the viewer-sinatra changes going live, since this is the same footer on all the site's pages regardless of which subdomain they're coming from (`docs.` or none).

### CODE REVIEW

**Comment by davewhiteland** [comment]
@zarino sanity check on this please, but seems to be lovely just by more-or-less dropping SASS and resources in from  everypolitician/viewer-sinatra#10691

I think the `one-third` column containing `mysoc-footer__legal` is staying unexpectedly thin when the page is narrow, but I'm happy to go live (with `viewer-sinatra` too) now and sort that out later if you give me a 👍

File: _layouts/default.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
   {% include head.html %}
 
-  <body>
+  <body class="dark-theme">
```
Comment (zarino) [inline _layouts/default.html]: Do you really need this? If you remove the `.dark-theme` selector from `_sass/_site-footer.scss`, then you won't need this class on the body.

File: _sass/_site-footer.scss [hunk 2]
```diff
@@ -1,46 +1,38 @@
...
+
+  $mysoc-footer-donate-background-color: #434343;
+  $mysoc-footer-donate-text-color: #fff;
+  $mysoc-footer-donate-button-background-color: $colour_red;
+
+  $mysoc-footer-legal-text-color: #9a9a9a;
+
+  $mysoc-footer-image-path: '../../images/mysoc-footer-dark/';
+
+  @import "ms-footer";
```
Comment (zarino) [inline _sass/_site-footer.scss]: Importing the footer styles here is fine, as long as you don't _also_ import them into `/css/main.scss`.

It's mostly stylistic preference, but I would put all of the `$mysoc-footer-…` override variables here (lines 1, 6–8, 10–21) into `/_sass/_variables.scss`, and then `@import "ms-footer"` just once, inside `/css/main.scss`. That way all the imports and all the variables are kept in the most obvious places.

Does that make sense?

Comment (zarino) [inline _sass/_site-footer.scss]: With all those variables and stuff gone, the only things left in this file will be your manual overrides of some of the footer styles: `.mysoc-footer .column-one-third` and `.mysoc-footer__links`. Tidy!

**Comment by zarino** [comment]
@davewhiteland – I'm happy with the width of `footer__legal` (we can always fix it later if it bothers anyone).

I've left a few comments on other bits though. Primarily: you were `@import`-ing the footer styles twice, resulting in twice as much compiled CSS as required. Removing the `dark-theme` class from the body and `_site-footer.scss` will remove the need for duplication. And while you're there, I'd move all the variables into `_variables.scss`, as explained in the other comment.

**Comment by davewhiteland** [comment]
@zarino ooh good stuff, thanks

**Comment by zarino** [comment]
👍  Much better! Thanks!

**Pull request merged**